packages/sitebuilder/src/utils/parseDomainListItem.ts

```diff
@@ -10,8 +10,8 @@ type DomainListItem = {
 	selected: boolean,
 }
 
-function getPrice(termFees: DomainTermFees, isAvailable: boolean) {
-	if (! isAvailable) {
+function getPrice(termFees: DomainTermFees | undefined, isAvailable: boolean) {
```

Comment:
No need to change it, but you can mark arguments as optional with "?" (see below)
```js
function getPrice(termFees?: DomainTermFees, isAvailable: boolean) {...}
```